src/components/Comment.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Comment.css';
+
+const Comment = ({ commentId, author, text, onDelete }) => {
+    return (
+        <div className="comment">
+            <div className="comment-content">
+                <span className="comment-author">{author}:</span>
+                <span className="comment-text">{text}</span>
```

Comment:
I'm suspecting that you're using span so that the two elements sit next to each other. However, the technically correct implementation would be for text to be in a text html element such as <p>.